backend/WebUI/api_webui.go

```diff
@@ -1358,6 +1358,42 @@ func dbOperation(ueId string, servingPlmnId string, method string, subsData *Sub
 	filterUeIdOnly := bson.M{"ueId": ueId}
 	filter := bson.M{"ueId": ueId, "servingPlmnId": servingPlmnId}
 
+	// Set Sd value to lower case
+	if subsData != nil {
```

Comment:
```
if subsData == nil {
    return
}
```

Reply:
It couldn't be returned because the function **DeleteSubscriberByID()** should call **dbOperation()**  with **argument subsData = nil** .